aptos-move/e2e-testsuite/src/tests/on_chain_configs.rs

```diff
@@ -25,7 +25,7 @@ fn initial_aptos_version() {
         println!("@@@@@@@@@@@@ account {:?}", account);
```

Comment:
Oh crap this looks like mine. Can you please remove it? I'm surprised lint didn't catch this.